libs/langchain/langchain/agents/load_tools.py

```diff
@@ -467,6 +471,8 @@ def _get_reddit_search(**kwargs: Any) -> BaseTool:
     ),
     "eleven_labs_text2speech": (_get_eleven_labs_text2speech, ["eleven_api_key"]),
     "google_cloud_texttospeech": (_get_google_cloud_texttospeech, []),
+  read_file
```

Comment:
typo here

Reply:
Can we solve this here or do you want me to create a new pull request?